pages/about.html

```diff
@@ -0,0 +1,203 @@
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse w-100" id="navbarSupportedContent">
+            <ul class="navbar-nav mb-2 mb-lg-0 align-items-center w-100 justify-content-lg-end justify-content-sm-evenly justify-content-evenly me-lg-5">
+              <li class="nav-item d-lg-none d-sm-block d-block">
+                <a class="nav-link navbar-brand w-100" href="/">
+                  <img src="../imgs/logo1.jpg" alt="logo" class="img-fluid"/>
+                </a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="/">Home</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link active" aria-current="page" href="./pages/about.html">About</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">contact</a>
+              </li>
+              <li class="nav-item me-lg-5 p-1 border border-success">
+                <a class="nav-link text-success" href="#">
+                  Log Out
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="container headline mt-4">
+      <nav aria-label="breadcrumb">
+        <ol class="breadcrumb justify-content-end me-70">
+          <li class="breadcrumb-item">
+            <a href="/" class="text-dark">
+              <i class="fas fa-home"></i>
+            </a>
+          </li>
+          <li class="breadcrumb-item active" aria-current="page">
+            About
+          </li>
+        </ol>
+      </nav>
+      <div class="content text-center mt-5">
+        <h1>
+          <span class="d-lg-none h2 d-block">"Hello Future Programmers"</span>
+          Web School
+        </h1>
+        <p class="m-auto">
+          We are a community of teachers and students. We work togther to create creative people who can
+          make awesome websites. So we have a special curriculum to make you professional web developer.
+          We will teach you the basics which are html, css, and js. After that you will learn framework for css
+          which is bootstrap 5. and a framework for js which is reactJS.
+        </p>
+        <div class="contact mt-5">
+          <h2 class="h6">
+            Please contact us if you have any further questions about web school!
+          </h2>
+          <a href="#" class="text-dark text-decoration-underline">
+            webscholl@gmail.com &raquo;
+          </a>
```

Comment:
- [ ] This section must have a border and a different background, please fix it.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132545276-f81d6e77-323a-4d75-a4b3-6abd7fea27d6.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132545366-eff3a2cf-12e0-4137-8271-a18f241f8933.png)

Reply:
It looks okay to me. Good job